libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/ImageCreator.java

```diff
@@ -27,26 +24,49 @@
  * If a shield URL is found, {@link Picasso} is used to load the image.  Then, once the image is loaded,
  * a new {@link ImageSpan} is created and set to the appropriate position of the {@link Spannable}
  */
-public class ImageCoordinator {
+public class ImageCreator extends NodeCreator<BannerComponentNode, ImageVerifier> {
```

Comment:
Should we double check / scope any `public` methods here?  I think `prefetch` and `shutdown` can stay public so developers can cache upcoming shields based on the upcoming step (so they load right away)

Reply:
@danesfeder Let me know if this looks good.